BoardTests.java

```diff
@@ -12,33 +14,44 @@ public void boardIsCreated() {
 
     @Test
     public void canFindWinnerXrow() {
-        Board board = new Board("---xxx---");
-        assertThat(board.checkForWin('x'), is(true));
+        Board board = new Board("------xxx");
+        assertThat(board.checkRowsForWin('x'), is(true));
```

Comment:
If you can get rid of checkRowsForWin then, checkRowsForWin could be replaced with getWinner() and assert that it's X..